editor/src/command.rs

```diff
@@ -109,6 +110,16 @@ impl<'l> From<TextNavCmd> for Command<'l> {
     }
 }
 
+impl<'l, T> From<T> for CommandGroup<'l>
+where
+    T: Into<Command<'l>>,
+{
+    fn from(cmd_like: T) -> CommandGroup<'l> {
+        let cmd: Command = cmd_like.into();
+        CommandGroup::Group(vec![cmd])
+    }
+}
+
```

Comment:
(If the other `impl` is a `From` as well, this is fine.)

Reply:
Only `From` is impl'd in that file, never `Into`. It seems common to use `Into` in trait bounds because it covers more things, but it doesn't matter here.